core/domain/user_services_test.py

```diff
@@ -642,6 +819,88 @@ def test_get_user_ids_by_role(self):
                 feconf.ROLE_ID_CURRICULUM_ADMIN)),
             set([user_ids[2], user_ids[3]]))
 
+    def test_update_user_bio(self):
+        auth_id = 'someUser'
+        user_email = 'user@example.com'
+        user_bio = 'new bio'
+
+        user_id = user_services.create_new_user(auth_id, user_email).user_id
+        user_services.update_user_bio(user_id, user_bio)
+        user_settings = user_services.get_user_settings_from_email(user_email)
+
+        self.assertEqual(user_bio, user_settings.user_bio)
```

Comment:
Missing pre-check to ensure that it wasn't user_bio before the main action of the test.